notifications_utils/s3.py

```diff
@@ -45,3 +45,60 @@ def s3download(bucket_name, filename):
         return key.get()["Body"]
     except botocore.exceptions.ClientError as error:
         raise S3ObjectNotFound(error.response, error.operation_name) from error
+
+
+S3_MULTIPART_UPLOAD_MIN_PART_SIZE = 5 * 1024 * 1024  # 5MB minimum multi part upload size
+
+
+def s3_multipart_upload_create(bucket_name, file_location, content_type="binary/octet-stream"):
+    s3 = client("s3")
+
+    args = {"Bucket": bucket_name, "Key": file_location, "ServerSideEncryption": "AES256", "ContentType": content_type}
+
+    try:
+        response = s3.create_multipart_upload(**args)
+        return response
+    except botocore.exceptions.ClientError as e:
+        current_app.logger.error("Unable to create multipart upload to S3 bucket %s", bucket_name)
+        raise e
+
+
+def s3_multipart_upload_part(part_number, bucket_name, filename, upload_id, data_bytes):
+    s3 = client("s3")
+
+    try:
+        response = s3.upload_part(
+            Bucket=bucket_name,
+            Key=filename,
+            PartNumber=part_number,
+            UploadId=upload_id,
+            Body=data_bytes,
+        )
+        return response
+    except botocore.exceptions.ClientError as e:
+        current_app.logger.error("Unable to upload part %s to S3 bucket %s", part_number, bucket_name)
```

Comment:
i think this log should include the filename too. maybe the upload ID as well, i'm not sure.